delivery.yaml

```diff
@@ -31,6 +31,9 @@ pipeline:
         echo "Running go mod tidy modified go.mod and/or go.sum"
         exit 1
       fi
+      for patch in patches/*.patch; do
+        cd e2e_modules/kubernetes && git apply "../../$patch" && cd -
```

Comment:
The usage of e2e_modules was dropped in https://github.com/zalando-incubator/kubernetes-on-aws/pull/7704